### DISCUSSION TITLE
How do I submit an issue on github itself category?

### DISCUSSION THREAD

**Discussion by kleag** [community/community#43487] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I think I have spotted an issue on the GitHub public GUI itself. Where should I submit it?

**Comment by radar-blip** [comment, marked as answer]
You can post in the [Issues](https://github.com/orgs/community/discussions/categories/issues) category describing the issue, someone might have a fix for it.
Or you can file a bug-report here: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by kleag** [reply]
Thank you. I did not find the bug report link.
